src/containers/ListingPage/ListingPage.js

```diff
@@ -264,6 +229,11 @@ export class ListingPageComponent extends Component {
       </span>
     );
 
+    const bookingTitle = (
+      <FormattedMessage id="ListingPage.bookingTitle" values={{ title: richTitle }} />
```

Comment:
It might make sense to 
- remove unnecessary span (`<FormattedMessage>`) and 
- use `intl.formatMessage({ id: 'ListingPage.bookingSubTitle' }, { title: richTitle });` instead.